termui/bug_table.go

```diff
@@ -290,10 +290,19 @@ func (bt *bugTable) render(v *gocui.View, maxX int) {
 	columnWidths := bt.getColumnWidths(maxX)
 
 	for _, excerpt := range bt.excerpts {
-		summaryTxt := fmt.Sprintf("C:%-2d L:%-2d",
-			excerpt.LenComments,
-			len(excerpt.Labels),
-		)
+		summaryTxt := fmt.Sprintf("%4d 💬", excerpt.LenComments)
+		labelsTxt := "" // fmt.Sprintf("L:%-2d", len(excerpt.Labels))
+
+		nbLabels := 0
+		for _, l := range excerpt.Labels {
+			lc := l.Color()
+			lc256 := lc.Term256()
+			labelsTxt += lc256.Escape() + " ◼" + lc256.Unescape()
+			nbLabels++
```

Comment:
you can use the first value of the `for` for that:
```
for i, l := range excerpt.Labels {
    if i > 5 {
        ....
```

Reply:
Yes, I could, but I need the value of `nbLabels` later in the code.
What's the best way to keep the value of the last `i` out of the `for` loop?

Reply:
ha yes i didn't notice that. The way you do it is good. Another way would be to have the next part of the code based on the resulting string instead of a counter but either way is fine.